SportsResults.frockett/SportsResults.frockett/EmailService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Net;
+using System.Net.Mail;
+
+namespace SportsResults.frockett;
+
+internal class EmailService
+{
+    // SMPT Address and port number are examples
+    static string smptAddress = "smtp.gmail.com";
+    static int portNumber = 587;
+    static bool enableSSL = true;
+    static string emailFromAddress = "";
+    static string password = "";
+    static string emailToAddress = "";
+
+    public static void SendEmail(string emailBody)
+    {
+        using MailMessage mail = new MailMessage();
+        mail.From = new MailAddress(emailFromAddress);
```

Comment:
Email addresses not validated and your email service throws exception when invalid mail provided. How would you fix that :thinking: :blush:

![Screenshot from 2024-03-27 12-41-42](https://github.com/TheCSharpAcademy/CodeReviews.Console.SportsResults/assets/57032138/6c2c7f0d-1baf-4c82-9be8-b37a51784287)

Reply:
I added a try/catch block to handle the exception.